portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -78,9 +94,35 @@ public void entityMarshall(Entity entity) {
    *
    * @param entity the entity to unmarshall data from.
    */
-  public void entityUnMarshall(Entity entity) {
+  public void entityUnmarshall(Entity entity) {
+    commentPoster = (String) entity.getProperty("poster");
     commentText = (String) entity.getProperty("text");
     Double score = (Double) entity.getProperty("sentiment-score");
     sentimentScore = score.floatValue();
   }
+
+  /**
+   * Generates a comment from an entity
+   *
+   * @param entity the entity to generate the comment from
+   * @return the new comment with the entity's information.
+   */
+  public static Comment from(Entity entity) {
+    // Elegant wrapper for private constructor.
+    return new Comment(entity);
+  }
+
+  /**
+   * Generates an entity from a comment
+   *
+   * @return A freshly-created Comment entity with the current timestamp.
+   */
+  public Entity toEntity() {
+    Entity entity = new Entity("Comment");
+    // Fill timestamp
+    entity.setProperty("timestamp", System.currentTimeMillis());
+    // Fill rest of comment
+    entityMarshall(entity);
```

Comment:
This is the only place this function is used. I'd recommend removing `entityMarshall` and instead inlining its contents here.

Reply:
Will change in this PR.